plugins/javadoc/src/main/kotlin/javadoc/KorteJavadocRenderer.kt

```diff
@@ -206,93 +204,167 @@ class KorteJavadocRenderer(val outputWriter: OutputWriter, val context: DokkaCon
             "description" to node.description
         )
     }
 
-    private fun renderPropertyNode(node: JavadocPropertyNode): PageContent {
+    private fun renderPropertyNode(node: JavadocPropertyNode): TemplateMap {
         val (modifiers, signature) = node.modifiersAndSignature
         return mapOf(
-            "modifiers" to renderContentNode(modifiers),
-            "signature" to renderContentNode(signature),
-            "description" to renderContentNode(node.brief)
+            "modifiers" to htmlForContentNode(modifiers),
+            "signature" to htmlForContentNode(signature),
+            "description" to htmlForContentNode(node.brief)
         )
     }
 
-    private fun renderEntryNode(node: JavadocEntryNode): PageContent =
-        mapOf(
-            "signature" to renderContentNode(node.signature),
-        ) + node.contentMap
+    private fun renderEntryNode(node: JavadocEntryNode): TemplateMap {
+        return mapOf(
+            "signature" to htmlForContentNode(node.signature),
+            "brief" to node.brief
+        )
+    }
 
 
     //TODO is it possible to use html renderer?
-    private fun renderContentNode(node: ContentNode): String =
-        when(node){
-            is ContentGroup -> node.children.joinToString(separator = "") { renderContentNode(it) }
+    private fun htmlForContentNode(node: ContentNode): String =
+        when (node) {
+            is ContentGroup -> node.children.joinToString(separator = "") { htmlForContentNode(it) }
             is ContentText -> node.text
             is TextNode -> node.text
-            is ContentLink -> """<a href="TODO">${node.children.joinToString { renderContentNode(it) }} </a>"""
+            is ContentLink -> """<a href="TODO">${node.children.joinToString { htmlForContentNode(it) }} </a>"""
             else -> ""
         }
 
+    private fun renderJavadocContentNode(node: JavadocContentNode): TemplateMap = when (node) {
+        is TitleNode -> renderTitleNode(node)
+        is JavadocContentGroup -> renderJavadocContentGroup(node)
+        is TextNode -> renderTextNode(node)
+        is ListNode -> renderListNode(node)
+        else -> emptyMap()
+    }
+
+    private fun renderTitleNode(node: TitleNode): TemplateMap {
+        return mapOf(
+            "title" to node.title,
+            "version" to node.version,
+            "packageName" to node.parent
+        )
+    }
+
+    private fun renderJavadocContentGroup(note: JavadocContentGroup): TemplateMap {
+        return note.children.fold(emptyMap<String, Any?>()) { map, child ->
+            map + renderJavadocContentNode(child)
+        }
+    }
+
+    private fun renderTextNode(node: TextNode): TemplateMap {
+        return mapOf("text" to node.text)
+    }
+
+    private fun renderListNode(node: ListNode): TemplateMap {
+        return mapOf(
+            "tabTitle" to node.tabTitle,
+            "colTitle" to node.colTitle,
+            "list" to node.children
+        )
+    }
```

Comment:
I guess those can be written with `=` instead of `: Type { return ... }` ( `=` looks better IMHO)